### ISSUE TITLE
API QueueConfigs limitation when creating routing profiles

### ISSUE THREAD

**Issue by jjwilliams42** [aws-samples/amazon-connect-copy#41]
The CreateRoutingProfile and UpdateRoutingProfileQueues API endpoints for some reason have a hard limit on the number of QueueConfigs that can be passed:

https://docs.aws.amazon.com/connect/latest/APIReference/API_CreateRoutingProfile.html#connect-CreateRoutingProfile-request-QueueConfigs

https://docs.aws.amazon.com/connect/latest/APIReference/API_UpdateRoutingProfileQueues.html#connect-UpdateRoutingProfileQueues-request-QueueConfigs

```
Array Members: Minimum number of 1 item. Maximum number of 10 items.
```

Since a QueueConfig consists of both a Queue and a Channel, if you have 2 channels and 5 queues, that is about all you can do when using amazon-connect-copy.

Would be great if amazon-connect-copy would account for this and utilize the AssociateRoutingProfileQueues API endpoint:

https://docs.aws.amazon.com/connect/latest/APIReference/API_AssociateRoutingProfileQueues.html

Maybe if it detected more than 10 QueueConfigs, it would resort to using the AssociateRoutingProfileQueues API.

Thank you for this amazing repo! Took our CI/CD to the next level with Connect.

**Comment by kjacky** [comment]
Hi Josh,

Thank you for your report. The issue has been fixed in v1.3.2:

```
- Version 1.3.2
  - Fix AWS CLI max 10 queue-configs in `aws connect associate-routing-profile-queues`
  - Fix QueueName select in RoutingProfileQueueConfigSummaryList
  - Remove routing_profile_to_ignore from connect_diff
  - Explicit with `.` in `jq -s '.'` (instead of just `jq -s`) in connect_copy
```

Could you download the latest v1.3.4 from main (or the beta v1.3.5 from development) to see if the 10 QueueConfigs limit has been fixed?

Jacky

**Comment by jjwilliams42** [comment]
Thank you! I guess I missed this.

You have no idea how much this made my day :D
